### PR TITLE
Adding utility method to check any protection is enforced

### PR DESCRIPTION
**Pull request by prasad-babu**
This utility method is useful if it is to check only whether document is
enforced (any type) or not

### CODE REVIEW

**Comment by centic9** [comment]
Any chance of a small unit-test together with a sample document (or using one of the available sample-documents) which covers the new function?

**Comment by prasad-babu** [comment]
Added test case with sample document

**Comment by prasad-babu** [comment]
sorry, i accidentally closed the request.

File: src/ooxml/java/org/apache/poi/xwpf/usermodel/XWPFTableCell.java [hunk 1]
```diff
@@ -233,8 +233,9 @@ public XWPFVertAlign getVerticalAlignment() {
         XWPFVertAlign vAlign = null;
         CTTcPr tcpr = ctTc.getTcPr();
         if (tcpr != null) {
-            CTVerticalJc va = tcpr.getVAlign();
-            vAlign = stVertAlignTypeMap.get(va.getVal().intValue());
+            CTVerticalJc va = tcpr.getVAlign();
+			if(va != null)
```
Comment (centic9) [inline src/ooxml/java/org/apache/poi/xwpf/usermodel/XWPFTableCell.java:237]: Is this change related? Looks like a different change/fix

**Comment by centic9** [comment]
FYI, I left out the seemingly unrelated change to XWPFTableCell.java, please create a separate PR with a short description and unit-test which reproduces the problem that you try to fix with it if you think this should be applied as well.

**Comment by prasad-babu** [comment]
Sorry that was different bug fix. Unfortunately it was added to this pull request.
